core/platform/android/java/src/org/axmol/lib/AxmolEngine.java

```diff
@@ -50,6 +50,7 @@ of this software and associated documentation files (the "Software"), to deal
 import android.util.Log;
 import android.view.Display;
 import android.view.DisplayCutout;
+import android.view.HapticFeedbackConstants;
```

Comment:
```suggestion
```

Reply:
Do you want me to `git commit --amend` that so I don't pollute commit history or should I just create regular commit?

Reply:
I commit it